### ISSUE TITLE
Unable to find terraform.exe: exec: "terraform.exe": executable file not found in %PATH%

### ISSUE THREAD

**Issue by jeremietharaud** [hashicorp/terraform-ls#152]
Hi,

The latest version of the extension (2.0.1) seems to crash immediately when starting VSCode with the following error:
```
Launching language server: c:\Users\jtharaud\.vscode\extensions\hashicorp.terraform-2.0.1/lsp/terraform-ls serve
2020/06/11 00:02:33 service.go:63: Preparing new session ...
2020/06/11 00:02:33 langserver.go:79: Starting server (pid 2580) ...
2020/06/11 00:02:33 server.go:469: Received 1 new requests
2020/06/11 00:02:33 server.go:165: Processing 1 requests
2020/06/11 00:02:33 server.go:223: Checking request for "initialize": {"process...
2020/06/11 00:02:33 rpc_logger.go:45: Error for "initialize" (ID 0): [-32098] unable to find terraform.exe: exec: "terraform.exe": executable file not found in %PATH%
2020/06/11 00:02:33 server.go:204: Completed 1 requests [15.0175ms elapsed]
[Error - 00:02:33] Server initialization failed.
  Message: unable to find terraform.exe: exec: "terraform.exe": executable file not found in %PATH%
  Code: -32098 
```

I'm running VSCode 1.46.0 on Windows 10 and I'm using tfenv for Terraform.

Any idea why this error appears ? Do you have a workaround ?

Thank you.

**Comment by radeksimko** [comment]
Probably related to #151

**Comment by dairappa** [comment]
> I'm not sure what the issue with `tfenv` would be, we have instructions on how to [set the Terraform executable manually](https://github.com/hashicorp/vscode-terraform/wiki/Manually-Setting-the-Terraform-Executable-Path) as a workaround though.

Thanks @paultyng , that workaround works well.
BTW, same issue occurs in Mac, also.

**Comment by jeremietharaud** [comment]
The issue seems to be fixed on version 2.1.1 of the extension, I don't reproduce the issue.
